front/lib/labs/config.ts

```diff
@@ -15,7 +15,7 @@ const config = {
           "NANGO_GOOGLE_DRIVE_CONNECTOR_ID"
         );
       case "gong":
-        return EnvironmentConfig.getEnvVariable("NANGO_GONG_CONNECTOR_ID");
```

Comment:
You can't really direclty go to a Nango removal. You have to support both flows temporarily until everything is migrated :+1: